docroot/modules/custom/yukon_migrate/migrations/nodes/yukon_migrate_landing_page_level_2.yml

```diff
@@ -0,0 +1,76 @@
+id: yukon_migrate_landing_page_level_2
+label: Landing Page Level 2
+migration_tags:
+    - Drupal 7
+    - node
+    - Landing Page Level 2
+migration_group: legacy
+source:
+    plugin: d7_node
+    node_type: landing_page_level_2
+    constants:
+        default_text_format: full_html
+destination:
+    plugin: entity:node
+    default_bundle: landing_page_level_2
+process:
+    uid: node_uid
+    nid: nid
+    title: title
+    status:
+        plugin: skip_on_value
+        method: row
+        not_equals: true
+        value: 1
+        source: status
+    langcode:
+        plugin: default_value
+        source: language
+        default_value: "en"
+    created: created
+    changed: changed
+    promote: promote
+    sticky: sticky
+    revision_log: log
+    field_page_description/value: field_page_description/0/value
+    field_page_description/format: constants/default_text_format
+    body/value: field_opener_description/0/value
+    body/format: constants/default_text_format
+    field_icon_dark:
+        plugin: migration_lookup
+        source: field_svg_upload/0/fid
+        migration: yukon_migrate_icon
```

Comment:
Do we need to add all of the migrations that are used in the migration lookup process to the migration dependencies?

Reply:
This has been resolved